src/ficamp/__main__.py

```diff
@@ -138,13 +143,25 @@ def categorize(engine):
         print("Session interrupted. Closing.")
 
 
+def sync(args, engine):
```

Comment:
I also find "sync" a bit confusing for this functionality. I would go for something more in the direction of "print-summary".
So that can later be the 1st step in exporting to a sink or a making a true synchronization.